crates/bevy_transform/src/hierarchy/child_builder.rs

```diff
@@ -39,37 +42,37 @@ pub struct PushChildren {
     children: SmallVec<[Entity; 8]>,
 }
 
-pub struct ChildBuilder<'a> {
-    commands: &'a mut Commands,
+pub struct ChildBuilder<'a, 'b> {
+    commands: &'a mut Commands<'b>,
```

Comment:
Could `Commands` become a type that can be cloned? Semantically kind of like an `Arc<Mutex<&mut Vec<Command>>>`? That would allow writing this as `commands: Commands<'a>`.

Reply:
I believe this was default behavior before 0.3 or 0.4, but it got removed to avoid unnecessry locking operations